apps/kindergarten/models.py

```diff
@@ -1 +1,30 @@
 # Create your models here.
+from django.db import models
+
+from core.models.helpers import image_path
+
+
+class Kindergarden(models.Model):
+    name=models.CharField(max_length=200)
+    image=models.ImageField(
+        upload_to=image_path,
+        blank=True
+    )
+    disctict=models.CharField(max_length=50)
```

Comment:
in the future we should create FK to location.City and location.District

Reply:
word mistake is found)

Reply:
I fixed mistakes, but I don't understand what to do with "in the future we should create FK to location.City and location.District" ?